src/main/java/WayofTime/alchemicalWizardry/BloodMagicConfiguration.java

```diff
@@ -398,12 +398,6 @@ public static void syncConfig() {
         AlchemicalWizardry.lpPerSelfSacrifice = config
                 .get("sacrifice", "LP per self-sacrifice", AlchemicalWizardry.lpPerSelfSacrifice)
                 .getInt(AlchemicalWizardry.lpPerSelfSacrifice);
-        AlchemicalWizardry.lpPerSelfSacrificeSoulFray = config
-                .get(
-                        "sacrifice",
-                        "LP per self-sacrifice (when Soul Fray potion is active)",
```

Comment:
Nope, you have to keep being able to sacrifice during soul fray, the 1/10th of the value was already a perfect fix as it was